inc/cartridgeitem.class.php

```diff
@@ -129,12 +129,12 @@ function defineTabs($options=array()) {
    *
    * @return number of cartridges
    **/
-   static function getCount() {
+   static function getCount($id) {
```

Comment:
Well... As using $this in a static call may have sort of unexpected behavior... No, I do not ;)